v1/ao/client_layer.go

```diff
@@ -44,10 +44,17 @@ func BeginRemoteURLSpan(ctx context.Context, spanName, remoteURL string, args ..
 // BeginRPCSpan returns a Span that reports metadata used by AppOptics to filter RPC call
 // latency heatmaps and charts by span name, protocol, controller, and remote host.
 // Call or defer the returned Span's End() to time the call's client-side latency.
-func BeginRPCSpan(ctx context.Context, spanName, protocol, controller, remoteHost string, args ...interface{}) Span {
-	rsKVs := []interface{}{"IsService", true,
-		"RemoteProtocol", protocol, "RemoteHost", remoteHost, "RemoteController", controller}
+func BeginRPCSpan(ctx context.Context, spanName, protocol, controller, remoteHost string,
+	args ...interface{}) Span {
+	rsKVs := []interface{}{
```

Comment:
Not related to this but I am wondering if we should use the preferred "single-key spec" as in https://github.com/librato/trace/blob/master/docs/specs/KV/remote-service-call.md#remote-service-call ?

Otherwise this looks good to me!

Reply:
Good point but I have no idea. @cheempz Is it worthwhile to switch to the `preferred` style? e.g., will the old spec be phased out recently?

Reply:
Looks like there is the `BeginRemoteURLSpan()` function above this one starting line 37 that lets the customer report using the preferred single-key spec.  So maybe all we need to do is add the `rsc` Spec KV to that too.